lib/vmdb/initializer.rb

```diff
@@ -5,55 +5,18 @@ def self.init
 
       Vmdb::Loggers.apply_config
 
-      if MiqEnvironment::Process.is_web_server_worker?
-        require 'hamlit-rails'
-
-        # Make these constants globally available
-        ::UiConstants
-      end
-
       # When these classes are deserialized in ActiveRecord (e.g. EmsEvent, MiqQueue), they need to be preloaded
       require 'VimTypes'
 
-      ####################################################
-      # If UiWorker called in Development Mode
-      #   invoked via command line -- script/server
-      #   invoked via debugger     -- using rdebug-ide gem
-      #
-      # Then, set up VMDB as if called from MiqServer:
-      #   1. SEED the MUST-HAVE classes
-      #   2. Mark current server as started
-      #
-      ####################################################
-      if MiqEnvironment::Process.is_ui_worker_via_command_line?
+      # UiWorker called in Development Mode
+      #   * command line(rails server)
+      #   * debugger
+      if defined?(Rails::Server)
```

Comment:
Replaced MiqEnvironment::Process call with a Rails constant check (so we can delete MiqEnvironment::Process)